File: server/src/main/java/io/microconfig/server/MicroconfigServer.java
package io.microconfig.server;

import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.scheduling.annotation.EnableScheduling;

import static org.springframework.boot.SpringApplication.run;

@EnableScheduling
@SpringBootApplication
public class MicroconfigServer {
    public static void main(String[] args) {
        run(MicroconfigServer.class, args);
    }
}

File: server/src/main/kotlin/io/microconfig/server/MicroconfigServer.kt
package io.microconfig.server

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.ConfigurationPropertiesScan
import org.springframework.boot.runApplication
import org.springframework.scheduling.annotation.EnableScheduling

@EnableScheduling
@SpringBootApplication
@ConfigurationPropertiesScan
class MicroconfigServer

fun main(args: Array<String>) {
    runApplication<MicroconfigServer>(*args)
}
